content_schemas/formats/content_block_tax_rate.jsonnet

```diff
@@ -0,0 +1,24 @@
+(import "shared/default_format.jsonnet") + {
+  document_type: "content_block_tax_rate",
+  base_path: "forbidden",
+  routes: "forbidden",
+  rendering_app: "forbidden",
+  definitions: {
+    details: {
+      type: "object",
+      additionalProperties: false,
+      required: ["band", "income", "rate"],
+      properties: {
+        band: {
+          type: "string",
+        },
+        income: {
+          type: "string",
+        },
+        rate: {
+          type: "string",
+        }
```

Comment:
This is an interesting one, because I think we're mixing up metadata and actual data here. To me, the `band` and the `income` would be metadata here, as they wouldn't actually be returned to the user. My mental model was more like we'd have two content types, `tax_income` and `tax_rate`, with individual body fields.

To use the example of a table of tax rates, I'd expect it to be used like this:

Band 	| Taxable income 	| Tax rate |
---------|----|----|
Personal Allowance | {{embed:tax_income:personal-allowance}} | {{embed:tax_rate:personal-allowance}}
Basic rate | {{embed:tax_income:basic-rate}} | {{embed:tax_rate:basic-rate}}
Higher rate | {{embed:tax_income:higher-rate}} | {{embed:tax_rate:higher-rate}}
Additional rate | {{embed:tax_income:additional-rate}} | {{embed:tax_rate:additional-rate}}

That said (and this is a thing that's just occurred to me), if we went with this approach we _could_ potentially have a more three-dimensional view and do something like this:


Band 	| Taxable income 	| Tax rate |
---------|----|----|
Personal Allowance | {{embed:tax_rate:personal-allowance:income}} | {{embed:tax_rate:personal-allowance:rate}}
Basic rate | {{embed:tax_rate:basic-rate:income}} | {{{embed:tax_rate:basic-rate:rate}}
Higher rate | {{embed:tax_rate:higher-rate:income}} | {{embed:tax_rate:higher-rate:rate}}
Additional rate | {{embed:tax_rate:additional-rate:income}} | {{embed:tax_rate:additional-rate:rate}}

Which could be quite cool. I think we'd need to run this via Rik first, as well as think how this would work within the rest of the user journey. Potentially super powerful though, and would also scale to things like contacts etc too

Reply:
hmm, I just thought that that entire row on this page for example could be a bit of re-usable content in other Editions https://www.gov.uk/income-tax-rates 

Is there another block type that's less controversial we could use that would give us a schema with multiple fields? I think it would be really useful to have a 'straw man' to help us see how the different types work.

Reply:
How about telephone number?